src/ucm/bistro/bistro.c

```diff
@@ -45,6 +48,59 @@ static ucs_status_t ucm_bistro_protect(void *addr, size_t len, int prot)
     return UCS_OK;
 }
 
+void ucm_bistro_apply_locking(void *dst, const ucm_bistro_lock_t *bytes)
+{
+    uint32_t value;
+
+    if (sizeof(value) > sizeof(*bytes)) {
```

Comment:
maybe remove this condition to simplify the code?

Reply:
I mean, always copy previous bytes to value without size check

Reply:
i see, fixed